src/registry/Registry.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * The registry that will have the singleton
+ * - getRegistry() -> the registry which is a simple key/value POJO
+ * @type {Object}
+ */
+const Registry = {
```

Comment:
So if this becomes a singleton, how about using multiple forms using different registries in a same page?